packages/ab-testing/package.json

```diff
@@ -1,18 +1,18 @@
 {
```

Comment:
We need to add the license in our `package.json` files too

Reply:
added